login.php

```diff
@@ -59,7 +59,7 @@
 		<div class="navigation">
 			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
-				<a href = "#"><img src = "images/logo_test2.png" height = "100"></a>
+				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
```

Comment:
24/9/2021: After some trialing and received some feedback I decided it is best to stick with the original logo that I create in when I develop the layout.